packages/react-jss/src/jss.js

```diff
@@ -1,5 +1,12 @@
 // @flow
-import {create} from 'jss'
+import {create, type JssOptions} from 'jss'
 import preset from 'jss-preset-default'
 
+export const createJss = (options?: Partial<JssOptions>) => {
```

Comment:
We could export the `create` function from JSS and the default presets, so one can combine both of them

Reply:
If wee do this, we don't really need to export `defaultJss` from JSS, since one could just use the create function.

Reply:
So I would just do this:

```js
export {create as createJss} from 'jss'
export {default as preset} from 'jss-preset-default'
```